sdk/src/loom/ModestMaps/core/Tile.ls

```diff
@@ -0,0 +1,123 @@
+/*
+ * vim:et sts=4 sw=4 cindent:
+ * $Id$
+ */
+
+package loom.modestmaps.core
+{
+    import loom2d.display.DisplayObject;
+    import loom2d.display.Sprite;
+    import loom2d.display.Image;
+    import loom2d.textures.Texture;
+    
+    
+    public class Tile extends Sprite
```

Comment:
Do you want me to come up with something? I checked any only 5 of the current 36 class/interface files have header documentation in them... it'll take a bit of time to go though and come up with something for all of them, but I can look into it as low priority in a week or two if you want?